### DISCUSSION TITLE
Journaling for AI

### DISCUSSION THREAD

**Discussion by morethanx** [brianpetro/obsidian-smart-connections#196] [Q&A]
hi there, I have been writing my journal since  late 2022 and I would like to ask GPT4 questions about it. For example "Today is 2023-05-06. Can you summarise the last two weeks" I also told them what my date format is, but they keep on summarising me some random days out of my journal and telling me that they do not have access. 

Does anyone know how to either teach GPT4 better how to do it or how I can have a folder system that they understand better.
My current folder system is: /Journal/2023/2023-05/W18/2023-05-06.

would be amazing if someone could give me some valuable tips. :)

thanks!

**Comment by brianpetro** [comment]
Hi @morethanx 

Currently, and based on your folder structure, the best way to do this would be to enter the relevant folders into the chat box. A "modal" (popup) should appear when you type a forward slash ("/") into the chat input, then you can select the relevant folder. You should be able to add multiple folders if necessary.

Also, since you're using GPT-4, you might want to increase the context window size, which would allow more people of your journal entries in a single chat input. You can see how to do that here https://github.com/brianpetro/obsidian-smart-connections/discussions/158#discussioncomment-5556382

Let me know if that helps!

Brian 🌴
